doc/index.rst

```diff
@@ -5,6 +5,15 @@ Demos
 
 This repository provides templates for the development of ros2_control-enabled robots and a simple simulations to demonstrate and prove ros2_control concepts.
 
+Repository organization
+^^^^^^^^^^^^^^^^^^^^^^^
+
+**Note:** the following list comprehensive.
```

Comment:
What is meant by this? Should this be
```suggestion
**Note:** the following list is comprehensive.
```
Or that it is not comprehensive, as the ``ros2_control_demo_bringup`` etc are not mentioned?